uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_manage-leave-requests.scss

```diff
@@ -57,20 +84,52 @@
  * @TODO remove this patch when Shoreditch theme is migrated to SSP
  */
 .chr_manage_leave_requests {
-  .btn.ui-select-toggle {
-    padding: 4px 10px !important;
-    text-transform: none !important;
+  &[data-role="admin"] {
+    .ui-select-toggle {
+      padding: 4px 10px !important;
+      text-transform: none !important;
+      box-shadow: none !important;
+    }
+
+    .form-group {
+      overflow: inherit !important;
+    }
+
+    .panel {
+      background-image: none !important;
+    }
+
+    .panel-body, .chr_vertical_tabs {
+      background-color: #f4f7f8;
+    }
+
+    input[type="search"] {
+      width: 100% !important;
+    }
+
+    .ui-select-toggle .btn-link i {
+      margin-top: 6px;
+    }
   }
 
-  .form-group {
-    overflow: inherit !important;
+  .btn-link {
+    border: none !important;
+    background-color: transparent !important;
   }
 
-  .panel {
-    background-image: none !important;
+  .chr_manage_leave_requests__sub-header .checkbox {
+    padding-top: 5px;
   }
+}
 
-  .panel-body, .chr_vertical_tabs {
-    background-color: #f4f7f8;
+/*
+ * This is a temporary patch which fixes CSS in the component (for SSP only)
+ * @TODO remove this patch when Shoreditch theme is migrated to SSP
+ */
+.page-manager-leave {
+  .chr_manage_leave_requests {
+    .btn-sm {
+      font-size: 100% !important;
```

Comment:
> .chr_manage_leave_requests__sub-header .checkbox

Otherwise not well aligned vertically and looks odd.

> .btn-sm

Otherwise in SSP looks smaller than other buttons, however in Admin (which has a priority because it uses Shoreditch) we must use btn-small according to the Invision and the Styleguide.